packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -48,6 +54,33 @@ export class BlobHandle implements IFluidHandle<ArrayBufferLike> {
     }
 }
 
+// todo: pipe real blob storage through from rehydrateContainer() (see #7721)
+class MockDetachedBlobStorage {
```

Comment:
Mock IMHO means test code, and there should be no text code in prod bits.
I think you mean something else here.
Also can you please add proper /** */ comments describing purpose of this class and usage?

Reply:
I tried to call this out in the comments/PR description, but I don't intend to check this in with this here. This should be a real `IDetachedBlobStorage` that comes from the host, so the blob contents can be persisted between containers. I would expect this to be done in #7721 and for this PR not to be checked in until that is done and use that storage instead. This shouldn't change any other part of the design of this PR.